core/tests/protractor_desktop/blogDashboard.js

```diff
@@ -0,0 +1,192 @@
+    await blogDashboardPage.navigateToBlogDashboardPage();
+    await blogDashboardPage.navigateToDraftsTab();
+    await blogDashboardPage.expectNumberOfDraftsBlogPostsToBe(0);
+
+    await blogDashboardPage.navigateToBlogDashboardPage();
+    await blogDashboardPage.navigateToPublishTab();
+    await blogDashboardPage.expectNumberOfPublishedBlogPostsToBe(1);
+
+    await blogDashboardPage.deleteBlogPostWithIndex(0);
+    await blogDashboardPage.expectNumberOfPublishedBlogPostsToBe(0);
+  });
+
+  it('should create, publish, unpublish and delete the blog post',
+    async function() {
+      await blogDashboardPage.createNewBlogPost();
+      await blogDashboardPage.navigateToBlogDashboardPage();
+      await blogDashboardPage.expectNumberOfDraftsBlogPostsToBe(1);
+
+      await blogDashboardPage.navigateToBlogPostEditorWithIndex(0);
+      await blogDashboardPage.publishBlogPost(
+        'Sample Blog Post', await forms.toRichText(
+          'Hi there, I’m Oppia! I’m an online personal tutor for everybody!')
+        , [1, 3, 5]);
+
+      await blogDashboardPage.navigateToBlogDashboardPage();
+      await blogDashboardPage.navigateToPublishTab();
+
+      await blogDashboardPage.unpublishBlogPostWithIndex(0);
+      await blogDashboardPage.expectNumberOfPublishedBlogPostsToBe(0);
+
+      await blogDashboardPage.navigateToDraftsTab();
+      await blogDashboardPage.expectNumberOfDraftsBlogPostsToBe(1);
+      await blogDashboardPage.deleteBlogPostWithIndex(0);
+      await blogDashboardPage.expectNumberOfDraftsBlogPostsToBe(0);
+    });
+
+  it('should create multiple blog posts both published and drafts and' +
+  ' check for navigation through list view', async function() {
+    await blogDashboardPage.createNewBlogPost();
+    await blogDashboardPage.saveBlogPostAsDraft(
+      'Sample Title1', await forms.toRichText(
+        'Hi there, I’m Oppia! I’m an online personal tutor for everybody!'));
+    await blogDashboardPage.navigateToBlogDashboardPage();
+
+    await blogDashboardPage.createNewBlogPost();
+    await blogDashboardPage.saveBlogPostAsDraft(
+      'Sample Title2', await forms.toRichText(
+        'Hi there, I’m Oppia! I’m a tutor for everybody!'));
+    await blogDashboardPage.navigateToBlogDashboardPage();
+
+    await blogDashboardPage.createNewBlogPost();
+    await blogDashboardPage.saveBlogPostAsDraft(
+      'Sample Title3', await forms.toRichText(
+        'Hi there, I’m Oppia! I’m a tutor for everybody here!'));
+    await blogDashboardPage.navigateToBlogDashboardPage();
+
+    await blogDashboardPage.expectNumberOfDraftsBlogPostsToBe(3);
+
+    await blogDashboardPage.getListView();
+    await blogDashboardPage.expectNumberOfBlogPostsRowsToBe(3);
+    await blogDashboardPage.navigateToBlogPostEditorWithIndexFromList(2);
+    var publishBlogPostButton = element(
+      by.css('.protractor-test-publish-blog-post-button'));
+    var confirmButton = element(
+      by.css('.protractor-test-confirm-button'));
```

Comment:
There should be no element selectors in the test files. Instead, all interactions with the page should go through the BlogDashboardPage utility

Reply:
Done.